catroid/src/main/java/org/catrobat/catroid/utils/CrashReporter.java

```diff
@@ -44,10 +42,8 @@
 
 	private static final String TAG = CrashReporter.class.getSimpleName();
 
-	public static final String RECOVERED_FROM_CRASH = "RECOVERED_FROM_CRASH";
-	public static final String EXCEPTION_FOR_REPORT = "EXCEPTION_FOR_REPORT";
-
 	private static SharedPreferences preferences;
+	public static final String EXCEPTION_FOR_REPORT = "EXCEPTION_FOR_REPORT";
 	private static boolean isCrashReportEnabled = BuildConfig.CRASHLYTICS_CRASH_REPORT_ENABLED;
```

Comment:
please remove this variable or make it all upper case, because this is hiding a constant behind a variable name.